hugging_stanza.py

```diff
@@ -82,52 +66,32 @@ def push_to_hub():
         # Create the repository
         repo_name = "stanza-" + model
         repo_id = "stanfordnlp/" + repo_name
-        repo_url = api.create_repo(
-            repo_id=repo_id,
-            token=HfFolder.get_token(),
-            exist_ok=True,
-        )
-
-        # Clone the repository
-        repo_local_path = os.path.join("hub", repo_name)
-
-        repo = Repository(repo_local_path, clone_from=repo_url)
-        # checkout "main" so that we know we are tracking files correctly
-        repo.git_checkout("main")
-        if not repo.is_repo_clean():
-            print(f"Repo {repo_local_path} is currently not clean.  Unwilling to proceed...")
-            break
-        repo.git_pull(rebase=True)
-
-        # Make sure jar files are tracked with LFS
```

Comment:
I would wonder a bit if missing this block for new repos will cause it to not use lfs for .zip or .pt files.  My memory of when we first set this up was that it did not put these kinds of files under lfs by default.

Reply:
... creating new repos now includes .zip and .pt ... thank you for the update!